core/comment.js

```diff
@@ -10,43 +10,42 @@
  */
 'use strict';
 
-goog.provide('Blockly.Comment');
-
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.Bubble');
-goog.require('Blockly.Css');
-goog.require('Blockly.Events');
+goog.module('Blockly.Comment');
+goog.module.declareLegacyNamespace();
+
+const Block = goog.requireType('Blockly.Block');
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const Bubble = goog.require('Blockly.Bubble');
+const Coordinate = goog.requireType('Blockly.utils.Coordinate');
+const Events = goog.require('Blockly.Events');
+const Icon = goog.require('Blockly.Icon');
+const Size = goog.requireType('Blockly.utils.Size');
+const Svg = goog.require('Blockly.utils.Svg');
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const {CommentModel} = goog.requireType('Blockly.Block');
+const {conditionalBind, Data, unbind} = goog.require('Blockly.browserEvents');
+const {dom, userAgent, object: utilsObject} = goog.require('Blockly.utils');
```

Comment:
I am not sure that this is allowed because these are all provided in different files.

In any case, my comment about destructuring was not about combining these to the same require, but rather, doing a similar process as when requireing `browserEvents`.

```
const {createSvgElement, HTML_NS} = goog.require('Blockly.utils.dom');
const {IE} = goog.require('Blockly.utils.userAgent');
const {inherits} = goog.require('Blockly.utils.object');
```

Reply:
Ah, got it (I think) - sorry for the confusion and thanks for clarifying!